### PR TITLE
Separate idle watchers for AC and Battery

### PR DESCRIPTION
**Pull request by adjamhub**
should fix #60

### CODE REVIEW

File: config/powermanagementsettings.cpp [hunk 1]
```diff
@@ -46,8 +46,10 @@ namespace PowerManagementSettingsConstants
     const QString POWER_LOW_LEVEL_KEY { QL1S("powerLowLevel") };
     const QString SHOW_ICON_KEY { QL1S("showIcon") };
     const QString USE_THEME_ICONS_KEY { QL1S("useThemeIcons") };
-    const QString IDLENESS_ACTION_KEY { QL1S("idlenessAction") };
-    const QString IDLENESS_TIME_SECS_KEY { QL1S("idlenessTimeSecs") };
+    const QString IDLENESS_AC_ACTION_KEY { QL1S("idlenessACAction") };
```
Comment (palinek) [inline config/powermanagementsettings.cpp:49]: One minor note.
In this new version we completely ignore the pre-existing "legacy" configuration.
IMO we should fallback to legacy configuration for both AC/baterry if there is no new configuration defined in the config yet.

E.g. something like
```cpp
return value(IDLENESS_AC_ACTION_KEY, value(IDLENESS_LEGACY_ACTION_KEY, -1)).toInt();
```

**Review by palinek — CHANGES_REQUESTED** [review]
Everything perfectly OK, but, please, implement the fallback mechanism in settings...

Thank you

File: config/powermanagementsettings.cpp (hunk 1 above)
Comment (adjamhub) [reply config/powermanagementsettings.cpp:49]: Ok.
I'm going to set the old values to be the actual settings of both AC/Battery watchers.

**Review by palinek — APPROVED** [review]
GTM

**Pull request merged**